source/SpinalHDL/Introduction/A simple example.rst

```diff
@@ -102,8 +102,8 @@ Finally, there is the component logic:
      io.flag := (counter === 0) | io.cond1
 
 ``counter`` is a register containing an 8-bits unsigned integer, with the
-initial value 0. Assignments to a registers can be read only after the next
-clock sampling.
+initial value 0. Assignments to change the state of a register can be read only
```

Comment:
initial value 0. Assignments to change the state of a register are available for read-back only